packages/models/src/resource-binding/manager.ts

```diff
@@ -0,0 +1,201 @@
+import { Transaction, Block, Header, Input, HexString, utils, Cell } from '@ckb-lumos/base'
+import { BI } from '@ckb-lumos/bi'
+import { Actor, ActorMessage, ActorURI, MessagePayload } from '..'
+import { TypeScriptHash, LockScriptHash } from './types'
+import { ResourceBindingRegistry, ResourceBindingManagerMessage } from './interface'
+import { outPointToOutPointString } from './utils'
+import { Listener } from '@ckb-js/kuai-io'
+import type { Subscription } from 'rxjs'
+import { ChainSource } from '@ckb-js/kuai-io/lib/types'
+import { OutPointString } from '../store'
+
+export class Manager extends Actor<object, MessagePayload<ResourceBindingManagerMessage>> {
+  #registry: Map<TypeScriptHash, Map<LockScriptHash, ResourceBindingRegistry>> = new Map()
+  #registryOutpoint: Map<OutPointString, ResourceBindingRegistry> = new Map()
```

Comment:
`#registryOutpoint` should be `#registerOutPoint` for `out point` in the camel case style